src/Jackett.Test/Definitions/DefinitionsParserTests.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Reflection;
+using Jackett.Common.Models;
+using NUnit.Framework;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+using Assert = NUnit.Framework.Assert;
+
+namespace Jackett.Test.Definitions
+{
+    [TestFixture]
+    public class DefinitionsParserTests
+    {
+        [Test]
+        public void LoadAndParseAllCardigannDefinitions()
+        {
+            var applicationFolder = Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath);
```

Comment:
I had to fix this bug in the updater script. Didn't realize it lived on in other parts of the code. relevant stackoverflow link. https://stackoverflow.com/questions/896572

```suggestion
            var applicationFolder = Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().EscapedCodeBase).LocalPath);
```

Reply:
done

Reply:
Wow, that's in a lot more places than I thought. TODO for later we need to combine these into a single function call.